MatrixKit/Models/MXKAppSettings.m

```diff
@@ -491,6 +493,30 @@ - (void)setEnableBubbleComponentLinkDetection:(BOOL)storeLinksInBubbleComponents
     }
 }
 
+- (NSArray<NSString *> *)firstURLDetectionIgnoredHosts
+{
+    if (self == [MXKAppSettings standardAppSettings])
+    {
+        return [NSUserDefaults.standardUserDefaults objectForKey:@"firstURLDetectionIgnoredHosts"] ?: @[[NSURL URLWithString:kMXMatrixDotToUrl].host];
```

Comment:
We can remove `?: @[[NSURL URLWithString:kMXMatrixDotToUrl].host];` as if someone wants to force all linked to be detected he can. `firstURLDetectionIgnoredHosts = nil` or `= @[]` can be accepted.